drivers/usbdev_synopsys_dwc2/usbdev_synopsys_dwc2.c

```diff
@@ -1021,10 +1021,35 @@ static void _usbdev_init(usbdev_t *dev)
 #endif
 }
 
+static usb_speed_t _get_max_speed(const usbdev_t *dev)
+{
+    dwc2_usb_otg_fshs_t *usbdev = (dwc2_usb_otg_fshs_t *)dev;
+    const dwc2_usb_otg_fshs_config_t *conf = usbdev->config;
+    if (IS_USED(DWC2_USB_OTG_HS_ENABLED) && (conf->type == DWC2_USB_OTG_HS)) {
+        return USB_SPEED_HIGH;
+    }
+    return USB_SPEED_FULL;
+}
+
+static usb_speed_t _get_enumeration_speed(const usbdev_t *dev)
+{
+    dwc2_usb_otg_fshs_t *usbdev = (dwc2_usb_otg_fshs_t *)dev;
+    const dwc2_usb_otg_fshs_config_t *conf = usbdev->config;
+    unsigned speed = (_device_regs(conf)->DSTS & USB_OTG_DSTS_ENUMSPD_Msk) >> USB_OTG_DSTS_ENUMSPD_Pos;
```

Comment:
```suggestion
    unsigned speed = (_device_regs(conf)->DSTS & USB_OTG_DSTS_ENUMSPD_Msk) >>
                     USB_OTG_DSTS_ENUMSPD_Pos;
```

Reply:
Squash it directly. Please rebase and include revert for PR #19437.

Reply:
Fixed, squashed and added two revert commits for the ones in #19437